sui/src/sui_commands.rs

```diff
@@ -380,11 +383,15 @@ impl SuiNetwork {
 
 pub async fn genesis(
```

Comment:
Should we separate the bench genesis from a clean genesis? It seems we are overreaching into production code quite a bit here to support bench.

Reply:
I pondered this too, but this means we will have a special validator flow for benchmarking and a different one for prod.

Is this a road we want to go down?

Reply:
we actually don't need this change in `genesis`, the `GenesisConfig` has a `keypair` struct that is supposed to represent the `single_address`

Reply:
after some thoughts, I think we could have different handling for normal genesis and benchmark genesis. They are all the same with the only exception of populating pre-generated objects. We do this by giving different genesis config.

Reply:
https://github.com/MystenLabs/sui/issues/1763